### PR TITLE
Add display data up5418

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
Adds support for OneAtlas display data.

In addition, removes deprecated functionality such as displaying order_ids and adds an additional filter to the catalog search to ensure only readily available scenes are returned.

### CODE REVIEW

**Review by chrieke — APPROVED** [review]
LGTM!

File: utils/widgets.py [hunk 1]
```diff
@@ -165,7 +165,9 @@ def create_search_params(start_date, end_date, sensors, max_cloudcover, limit):
                 sortby="cloudCoverage",
                 limit=limit.value,
             )
+            search_parameters["query"]["deliveryTime"] = {"in": ["MINUTES"]}
```
Comment (thaisbendixen) [inline utils/widgets.py:168]: What exactly does this do? Is this the part that "ensures only readily available scenes are returned"?

**Review by thaisbendixen — APPROVED** [review]
LGTM!! 💪

**Review by markusuwe — APPROVED** [review]
LGTM

File: utils/widgets.py (hunk 1 above)
Comment (rodrigoalmeida94) [reply utils/widgets.py:168]: Precisely - I got this from @iliascholl-up42 here https://up-42.slack.com/archives/CV9NPMRR7/p1616416876002900

**Pull request merged**